.github/workflows/phrase-export-checks.yml

```diff
@@ -0,0 +1,84 @@
+name: Check Phrase translations
```

Comment:
Maybe "Check Phrase exports" or "Check exported Phrase translations" just to make it a tiny bit clearer to new folks that it shouldn't be running on all translations they might've added?